Package/Installation.cs

```diff
@@ -86,7 +86,31 @@ public Installation(string directory)
         /// <summary>
         /// Get the installation of the currently running tap process
         /// </summary>
-        public static Installation Current => current ??= new Installation(ExecutorClient.ExeDir);
+        public static Installation Current
+        {
+            get
+            {
+                if (current == null)
+                {
+                    current = new Installation(ExecutorClient.ExeDir);
+                    current.PrintReadable();
```

Comment:
I think it is a nice addition, but also that we should add it a bit more deliberately.

Maybe add it together with 'SessionLog.SystemInfo'?

Reply:
Then we can also move the system info out of SessionLogs.

Reply:
I added an interface and moved the startup logging. Was this what you had in mind?